core/src/scene/sceneLoader.cpp

```diff
@@ -1341,10 +1341,10 @@ SceneLayer SceneLoader::loadSublayer(Node layer, const std::string& name, Scene&
                 std::vector<StyleParam> params;
                 parseStyleParams(ruleNode.second, scene, "", params);
 
-                auto name = ruleNode.first.as<std::string>();
-                int nameId = scene.addIdForName(name);
+                auto drawStyle = ruleNode.first.as<std::string>();
```

Comment:
Since https://github.com/tangrams/tangram-docs/pull/65 is in sight of making the documentation unambiguous, can we have this named to ruleName or something on those lines?